src/utils/helpers.ts

```diff
@@ -0,0 +1,55 @@
+import { createComment } from "./create-comment.js";
+
+const profilePicture =
+  "https://cdn.pixabay.com/photo/2021/05/20/11/58/beauty-6268460_1280.jpg";
+
+export const createNewElement = (
```

Comment:
This method also exists in src/utils/create-element.

why?

Reply:
deleted!